PWGHF/TableProducer/candidateCreatorSigmac0plusplusCascade.cxx

```diff
@@ -0,0 +1,324 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file candidateCreatorSigmac0plusplusCascade.cxx
+///
+/// \note Task for Σc0,++ → Λc(→ K0sP) + π-,+ analysis
+/// \note Here the Lc from the cascade channel is obtained using the task taskLcToK0sP.cxx
+/// \author Rutuparna Rath <rrath@cern.ch>, INFN BOLOGNA and GSI Darmstadt
+/// In collaboration with Andrea Alici <aalici@cern.ch>, INFN BOLOGNA
+
+#include "CommonConstants/PhysicsConstants.h"
+#include "Common/Core/TrackSelection.h"
+#include "Common/Core/trackUtilities.h"
+#include "Common/Core/TrackSelectionDefaults.h"
+#include "Framework/AnalysisTask.h"
+#include "Framework/runDataProcessing.h"
+#include "PWGHF/Core/HfHelper.h"
+#include "PWGHF/DataModel/CandidateReconstructionTables.h"
+#include "PWGHF/DataModel/CandidateSelectionTables.h"
+
+using namespace o2;
+using namespace o2::analysis;
+using namespace o2::constants::physics;
+using namespace o2::framework;
+using namespace o2::framework::expressions;
+
+struct HfCandidateCreatorSigmac0plusplusCascade {
+  Configurable<double> cfgtrkMinPt{"cfgtrkMinPt", 0.15, "track min pT"};
+  Configurable<double> cfgtrkMaxEta{"cfgtrkMaxEta", 0.8, "track max Eta"};
+  Configurable<double> cfgMaxDCAxyToPVcut{"cfgMaxDCAxyToPVcut", 2.0, "Track DCAxy cut to PV Maximum"};
+  Configurable<double> cfgMaxDCAzToPVcut{"cfgMaxDCAzToPVcut", 2.0, "Track DCAz cut to PV Maximum"};
+  Configurable<double> cfgnFindableTPCClusters{"cfgnFindableTPCClusters", 120, "nFindable TPC Clusters"};
```

Comment:
afaik currently this variable is not well defined in Run3, yous should switch to `found clusters` (they should be pretty equivalent)

Reply:
Thank you. Done.